### DISCUSSION TITLE
next/link force prepend defaultLocale

### DISCUSSION THREAD

**Discussion by andylemaire** [vercel/next.js#25037] [Ideas]
### Describe the feature you'd like to request

Hi,

As I understand it now, the locale is only being prepended on `next/link` tags `<Link href="/link">` when you're using a locale which is *not* the `defaultLocale`.

Would it be possible to add a feature to force prepending the `locale` even when you're browsing on the `defaultLocale`?

### Describe the solution you'd like

Option to force prepending the `locale` even when browsing the `defaultLocale`.

### Describe alternatives you've considered

Manually add the `locale` from `router` to the `<Link href="/${local}/link">`.

**Comment by jayant-lumino** [comment]
Is this possible today? As I have listing and I want the URL to be prefix locale

So this is how it's right now
http://example.com/questions/

But I want it to be
http://example.com/en/questions/
